sheet_mass: 0.5
sheet_width: 1.0
